packages/trader/package.json

```diff
@@ -41,7 +42,7 @@
   "main": "dist/src/index.js",
   "scripts": {
     "build": "tsc -b",
-    "test": "hardhat test"
+    "test": "hardhat test  --network hardhat"
```

Comment:
do you need to specify --network hardhat? isn't that default?

Reply:
no but this is how we do it in other tests :)

Reply:
> actually this is not in other tests. I've removed it.